osm2lanes/src/transform/tags_to_lanes/infer.rs

```diff
@@ -0,0 +1,117 @@
+use super::TagsToLanesMsg;
```

Comment:
Going forward, if there's a PR moving stuff around, can we _just_ do that step first? Much easier to review mechanical refactors separate from ones that change something interesting. Because git doesn't pick up the rename well, I don't really know if something in here changed

Reply:
nothing meaningful, but agreed, I will make more of the refactoring PRs I made before